### ISSUE TITLE
Crash when setting survey title after adding survey elements

### ISSUE THREAD

**Issue by Miltage** [surveyjs/survey-creator#696]
I have an issue where the survey creator crashes when trying to close the Survey Settings dialog. This only happens when using the editor modal by setting the option `showPropertyGrid: false` and when there are elements in the survey. No issues if I open the dialog and change settings before adding any elements.

```
editableObject.ts:54 Uncaught TypeError: Cannot set property pages of #<SurveyModel> which has only a getter
    at EditableObject.apply (editableObject.ts:54)
    at EditableObject.applyAll (editableObject.ts:62)
    at SurveyQuestionEditor.apply (questionEditor.ts:684)
    at SurveyQuestionEditor.doCloseWindow (questionEditor.ts:651)
    at SurveyQuestionEditor.self.onOkClick (questionEditor.ts:608)
    at HTMLInputElement.<anonymous> (knockout-latest.js:100)
```

Might have to resort back to using the property editor side panel.

My code
```
let options = { 
    showEmbededSurveyTab: false,
    showJSONEditorTab: false,
    generateValidJSON: true,
    showPropertyGrid: false,
    useTabsInElementEditor: true,
    questionTypes: ["text", "checkbox", "radiogroup", "dropdown", "comment", "boolean"]
};
this.surveyCreator = new SurveyJSCreator.SurveyCreator(
    "surveyCreatorContainer",
    options
);
```

**Comment by andrewtelnov** [comment]
@Miltage Please use the latest version. Here is the [related version](https://github.com/surveyjs/survey-creator/issues/674).
It was fixed in [v1.5.11](https://surveyjs.io/whatsnew)

Thank you,
Andrew

**Comment by Miltage** [comment]
Thanks Andrew. I did search for similar issues but think I made the mistake of searching only open issues.
